README.md

```diff
@@ -16,16 +16,58 @@ Why the need for other testing?  The OpenJDK regression tests are a great start,
 
 The test infrastructure in this repository allows us to lightly yoke a great variety of tests together to be applied to testing the AdoptOpenJDK binaries.  It is a thin wrapper around a varied set of tests, to allow us to easily run all types of tests via make targets and as stages in our Jenkins CI pipeline builds.
 
+Note: there are additional changes coming to the testing at AdoptOpenJDK (re: top level make targets and possibly even node labels in Jenkins), so we will be in flux for a little while. Stay in tune.
+
 # Running AdoptOpenJDK Tests
-Different platform: x64_linux | x64_mac | s390x_linux | ppc64le_linux | aarch64_linux
-Different jvmversion: openjdk9-openj9 | openjdk9 | openjdk8
-Different javaversion: SE80 | SE90
-## 0. Configure environment and get dependencies:
+Platform: x64_linux | x64_mac | s390x_linux | ppc64le_linux | aarch64_linux
+
+Javaversion: SE80 | SE90
+
+Prerequisites:
+	
+    perl 5.10.1 or above with JSON and Text::CSV module installed
+    make 3.81 or above (recommend make 4.1 or above on windows)
+    wget 1.14 or above
+    ant 1.7.1 or above with ant-contrib.jar
+
+## Jenkins setup and running
+AdoptOpenjdk set up test builds against adoptopenjdk sdk builds, release sdk, nightly sdk.
```

Comment:
While you can run all the tests manually via the make targets on the command line, you may also run the tests in Jenkins.  As part of the AdoptOpenJDK continuous integration (CI), AdoptOpenJDK runs test builds against the release and nightly SDK builds.